tests/c_languages/cppcheck_test_files/bad_file.cpp

```diff
@@ -0,0 +1,8 @@
+#define f(c) { 
+    char s[10];
+    s[c] = 42;
+}
```

Comment:
have you seen that the original file has escaped the `\n`, so the first `#define` directive appears only on the first line? (Otherwise it would be invalid C++ code anyway)

Reply:
@Makman2 I didn't get you.Can I change this file to some other example?

Reply:
>have you seen that the original file has escaped the \n, so the first #define directive appears only on the first line? (Otherwise it would be invalid C++ code anyway)

okay I got it :)
now I think I should escape these lines so that it appears only on the first line.